.husky/commit-msg

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
-npx commitlint --edit 
```

Comment:
We still need this to enforce the commit message conventions, which are used by semantic-release.

Reply:
Now that this is going to be generating code for more platforms, we are probably going to have to rethink our conventions for semantic-release (should a breaking change to the Dart build really trigger a new release on the JS package? Probably not). But I think that for the time-being we should leave things as-is and update when we have a reasonable alternative.